wandb/sdk/launch/agent/agent.py

```diff
@@ -14,7 +14,7 @@
 from wandb.errors import CommError
 from wandb.sdk.launch.launch_add import launch_add
 from wandb.sdk.launch.runner.local_container import LocalSubmittedRun
-from wandb.sdk.launch.sweeps.scheduler import Scheduler
+from wandb.sdk.launch.sweeps.scheduler import RunState, Scheduler
```

Comment:
Probably want to refactor `RunState` out of `scheduler.py` if we are using it in the agent for all runs.